pom.xml

```diff
@@ -82,6 +82,11 @@
     	<artifactId>dbpool</artifactId>
     	<version>7.0-jdk7</version>
     </dependency>
+    <dependency>
+        <groupId>com.djdch.log4j</groupId>
+        <artifactId>log4j-staticshutdown</artifactId>
+        <version>1.1.0</version>
```

Comment:
this is a nice little jar that improves the behavior of log4j2 -- with this, you can actually log while the jvm is going down.